tests/selenium/dnt_test.py

```diff
@@ -31,6 +31,19 @@ class DNTTest(pbtest.PBSeleniumTest):
         #"privacy-badger-navigator-donottrack-fixture.html"
     )
 
+    def get_first_party_headers(self, url):
+        self.load_url(url)
+
+        text = self.driver.find_element_by_tag_name('body').text
+
+        try:
+            headers = json.loads(text)['headers']
+        except ValueError:
+            print("\nFailed to parse JSON from {}".format(repr(text)))
+            return False
```

Comment:
maybe this should return [] instead?

Reply:
Shouldn't we have a clear failure-to-get-headers scenario? An empty array (did you mean object?) suggests we got headers but they were empty ... I dunno, does this make sense or am I missing something? Also, is there a Python convention for failure return values? Like, should we return `None` instead?

Reply:
Oh yeah, I think you're right. That makes sense. Not sure what the convention in Python is, but returning False will get the job done.